src/styles/util/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin flex-between {
```

Comment:
It can be used as a placeholder.
Better add `square` mixin that accepts value as argument and return equal width and height (use with logo and star).